addons/website/static/tests/tours/font_familty.js

```diff
@@ -0,0 +1,94 @@
+            run: "click",
+        },
+        {
+            content: "Simulate file input selection",
+            trigger: "input#upload_font",
+            run: async function () {
+                async function fetchFileAndSimulateInput(filePath, fileInput) {
+                    try {
+                        const response = await fetch(filePath);
+                        const blob = await response.blob();
+                        const file = new File([blob], "Lato-BlaIta-webfont.woff", {
+                            type: "font/woff",
+                        });
+
+                        const dataTransfer = new DataTransfer();
+                        dataTransfer.items.add(file);
+                        fileInput.files = dataTransfer.files;
+                        fileInput.dispatchEvent(new Event("change", { bubbles: true }));
+                    } catch (error) {
+                        console.error("Error fetching the file:", error);
+                    }
+                }
+
+                // Fetch the file and simulate the input
+                var fileInput = this.anchor;
+                await fetchFileAndSimulateInput(
+                    "/web/static/fonts/lato/Lato-BlaIta-webfont.woff",
+                    fileInput
+                );
+            },
+        },
+        {
+            content: "Check that the preview of 'Lato-BlaIta-webfont' font is loaded",
+            trigger: "div[style*=\"font-family: 'Lato-BlaIta-webfont';\"]",
+        },
+        {
+            content: "Click on the 'Save and reload' button",
+            trigger: "button.btn-primary",
+            run: "click",
+        },
+        ...wTourUtils.goToTheme(),
+        {
+            content: "Verify that 'Lato-BlaIta-webfont' font family is applied",
+            trigger:
+                "we-select[data-variable='headings-font'] we-toggler[style*='font-family: Lato-BlaIta-webfont;']",
+        },
+        {
+            content: "Open the heading font family selector",
+            trigger:
+                "we-select[data-variable='headings-font'] we-toggler[style*='font-family: Lato-BlaIta-webfont;']",
+            run: "click",
+        },
+        {
+            content: "Click on the 'Add a custom font' button again",
+            trigger: "we-select[data-variable='headings-font'] .o_we_add_font_btn",
+            run: "click",
+        },
+        {
+            content: "Wait for the modal to open and then refresh",
+            trigger: "button.btn-secondary",
+            run: function () {
+                setTimeout(() => {
+                    this.anchor.click();
+                }, 2000);
+            },
+        },
+        {
+            content:
+                "Check that 'Lato-BlaIta-webfont' font family is still applied and not reverted",
+            trigger:
+                "we-select[data-variable='headings-font'] we-toggler[style*='font-family: Lato-BlaIta-webfont;']",
```

Comment:
Tour is too specific for this case, maybe we should write tour with generic case instead of following the same steps given in task, we should test reload/_refreshBundles should not be called when we select Add Custom Font ```we-button```

Reply:
Hello @msh-odoo "The current [steps](https://github.com/odoo/odoo/pull/190329/files#diff-7fc5f8398accb6d9385a64f6e66ef60342d824794f0e0cdbb474c9aa403eff27R68-R91) are designed specifically to test that reload/_refreshBundles are not called, as the 'Lato-BlaIta-webfont' font family remains applied and does not revert. This ensures these methods were not triggered, since triggering them would have caused the final validation step to fail."